pom.xml

```diff
@@ -284,6 +282,16 @@
       <artifactId>sac</artifactId>
       <version>1.3</version>
     </dependency>
+    <dependency>
```

Comment:
@ctubbsii I'm curious to what issue this is attempting to fix.  The build already works on jdk 11.  Was there something specific you were seeing?

Reply:
@hazendaz  The main thing is the use of the `-release` option, which replaces the need to use `-bootstrapclasspath` or something like `animal-sniffer`. Since we didn't actually have any linking to newer APIs that would have made it break on Java 8, it probably didn't matter yet... but this adds future-proofing by using the `-release` / `maven.compiler.release` option to enforce compiler compliance when cross-compiling to the older version of Java.

The use of the `xml-apis` jar was a necessary stupid workaround, because JDK11 is dumb and is stricter than JDK8 was for Java 8 compliance. We probably could list that dependency as "optional", so that it's "compiler-time only" (because the JRE provides it at runtime) but I'm not sure how "optional" deps get resolved for Maven plugins. It shouldn't matter, though. I think it's fine to leave this extra dependency in.

Reply:
For more details about the xml-apis thing: In Java 8, org.w3c.dom.css was *NOT* part of the JAXP specification, but it was still made available and included in Java 8, as though it were. This was fixed after Java 8 by expanding the specification to include org.w3c.dom.css. Essentially, the specification was broken, not the code. So, in Java 8 worked, even though it wasn't part of the specification, because it should have been part of it. In Java 11, it works because it is now part of the specification and part of the xml module. But, it's broken when JDK11 tries to cross-compile to Java 8, because it's enforcing the previous, never before strictly enforced, broken specification.